### ISSUE TITLE
JSON Data for Objects - Terrain

### ISSUE THREAD

**Issue by ctomni231** [ctomni231/cwtactics#473]
Custom Wars Design Document - [2.0] Terrain
https://github.com/ctomni231/cwtactics/wiki/Custom-Wars-Tactics---Core-Document#20-terrain

We need a data driven way to represent terrain tiles within the system. We've pretty much already accomplished this before in CWT Alpha 0.35. You can see this here:
https://github.com/ctomni231/cwtactics/blob/gh-pages/0_3_5/mod/cwt/tiles.json

A similar structure for the updated version will be helpful. We can either choose to store this in code, traits, or as separated JSON files.

- [ ] An updated file containing data needed for terrain tiles

**Comment by Tapsi** [comment]
I would like to separate the issue. this may introduce the leading mechanism only. :)

**Comment by ctomni231** [comment]
Okay, so in other words, we are aiming for functionality rather than data storage.

**Comment by Tapsi** [comment]
both but if we add data now, then we have to support it. e.g. we have to change all data when we change the data structure. cause of this I would suggest to add only the functionality plus one or two test tiles/units only. :) 
what do you think ?

**Comment by ctomni231** [comment]
Yeah, a small subset will be great. I only want us to start thinking about the structure, because it'll get messy if we don't start early.

**Comment by Tapsi** [comment]
yeah it's probably a good time to add data structures, since we have a map now
